tests/test_slim_engine.py

```diff
@@ -3079,6 +3079,79 @@ def test_errors(self):
             stdpopsim.selection_coeff_from_mutation(ts, "bar")
 
 
+@pytest.mark.skipif(IS_WINDOWS, reason="SLiM not available on windows")
```

Comment:
I don't know the stdpopsim code at all, but just here kibitzing, to say: SLiM *is* available on Windows... :->

Reply:
That's a different issue that is open still because getting temp files to work correctly in Windows is a Pain.